newdle/client/src/components/Summary.js

```diff
@@ -0,0 +1,80 @@
+import React, {useEffect, useState} from 'react';
+import {serializeDate, toMoment} from '../util/date';
+import {useDispatch, useSelector} from 'react-redux';
+import {Button, Container, Icon, Header} from 'semantic-ui-react';
+import ParticipantTable from './ParticipantTable';
+import {clearFinalDate} from '../actions';
+import {getNewdle, getFinalDate} from '../selectors';
+import {updateNewdle} from '../actions';
+import client from '../client';
+import styles from './Summary.module.scss';
+
+export default function Summary() {
+  const [submitting, setSubmitting] = useState(false);
+  const [error, setError] = useState('');
+  const newdle = useSelector(getNewdle);
+  const finalDate = useSelector(getFinalDate);
```

Comment:
I don't think we need this in redux at all. It's never used outside this component, so it could stay in local state